src/components/filter/Filter.tsx

```diff
@@ -0,0 +1,38 @@
+import { useContext, SetStateAction } from "react";
+import * as Styled from "./Filter.styled";
+import { TeamEntriesContext } from "../../context/TeamEntriesProvider";
+
+interface FilterProps {
+  setCurrentClient: React.Dispatch<SetStateAction<string>>;
+}
+
+export const Filter = ({ setCurrentClient }: FilterProps) => {
+  const { teamEntries, setTeamEntries } = useContext(TeamEntriesContext);
+
+  const handleFilter = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    setCurrentClient(event.target.value);
+  };
+
+  const onlyClients = teamEntries.map(function (entry) {
+    return entry.client;
+  });
```

Comment:
You can shorten this function to 
```suggestion
  const onlyClients = teamEntries.map((entry) => entry.client);
```

Reply:
I'd also change the name of your variable to something like `clients`; `onlyClients` feels a little verbose in my opinion (and also a little like OnlyFans 😜).